tox.ini

```diff
@@ -11,10 +11,18 @@ changedir = tmp
 extras =
   tests
   qt
+  docs
```

Comment:
```suggestion
```

Reply:
You reckon we should keep the docs build separate from the other ones when running `tox` at root?

Reply:
I am not sure I follow.